packages/kbn-plugin-generator/sao_template/sao.js

```diff
@@ -89,6 +89,7 @@ module.exports = function({ name }) {
           kebabCase,
```

Comment:
Is there any reason to leave `kebabCase` here? Can we get rid of it and the `lodash.kebabcase` module?

Reply:
You are right, I've also checked this and it's no more used. @aaronoah could you check again and in case remove that line?

Reply:
done.